src/MongoDB/Query.c

```diff
@@ -228,6 +228,23 @@ HashTable *php_phongo_query_get_debug_info(zval *object, int *is_temp TSRMLS_DC)
 	ADD_ASSOC_LONG_EX(&retval, "limit", intern->limit);
 	ADD_ASSOC_LONG_EX(&retval, "batch_size", intern->batch_size);
 
+	if (intern->read_concern) {
+#if PHP_VERSION_ID >= 70000
+		zval read_concern;
+
+		php_phongo_read_concern_to_zval(&read_concern, intern->read_concern);
+		ADD_ASSOC_ZVAL_EX(&retval, "readConcern", &read_concern);
```

Comment:
Does ADD_ASSOC_ZVAL_EX copy the read_concern zval? If not, it's value is going to be out of scope when the function ends. I would at least run the tests covering this with valgrind.